README.adoc

```diff
@@ -168,6 +170,13 @@ so it can be used later by the `system` and `inventory` services:
 mvn -pl models install
 ```
 
+If you have not pulled the latest Docker image yet, do so by running the following command:
```

Comment:
Suggested update for line 173: 
"If you don't have the latest Docker image, pull it by running the following command:"

Reply:
Fixed with latest commit, thanks @Charlotte-Holt!